tests/std/tests/P0896R4_P1614R2_comparisons/test.cpp

```diff
@@ -465,9 +456,14 @@ constexpr void ordering_test_cases() {
     test_partially_ordered(1.414f, 3.14, partial_ordering::less);
     test_partially_ordered(1.414, 3.14f, partial_ordering::less);
     test_partially_ordered(31.625f, 31.625, partial_ordering::equivalent);
-    test_partially_ordered(3.14, NaN, partial_ordering::unordered);
-    test_partially_ordered(3.14f, NaN, partial_ordering::unordered);
-    test_partially_ordered(3.14, NaNf, partial_ordering::unordered);
+#if !defined(__clang__) && !defined(__EDG__) // TRANSITION, GH-741
+    if (!std::is_constant_evaluated())
+#endif // TRANSITION, GH-741

```

Comment:
I filed a "NaN is broken in constexpr" bug when I wrote this test, but didn't annotate it here for no reason I can recall:
```suggestion
#if !defined(__clang__) && !defined(__EDG__) // TRANSITION, VSO-1062601
    if (!std::is_constant_evaluated())
#endif // TRANSITION, VSO-1062601
```